test-it/testList.json

```diff
@@ -31,7 +31,8 @@
         },
         "manual": {
             "tests": [
-                "StringFormatTime.via"
+                "StringFormatTime.via",
+                "TestVI_LargeBDTest.via"
```

Comment:
Nothing fails when the test is included, but the fix is. My current changeset is adding the TestVI_LargeBDTest.via to the manual tests.